test/Microsoft.AspNet.Mvc.Extensions.Test/JsonResultTest.cs

```diff
@@ -20,8 +21,11 @@ public class JsonResultTest
         private static readonly byte[] _abcdUTF8Bytes
             = new byte[] { 123, 34, 102, 111, 111, 34, 58, 34, 97, 98, 99, 100, 34, 125 };
 
+        // Newlines dont have /r in mono.
         private static readonly byte[] _abcdIndentedUTF8Bytes
-            = new byte[] { 123, 13, 10, 32, 32, 34, 102, 111, 111, 34, 58, 32, 34, 97, 98, 99, 100, 34, 13, 10, 125 };
+            = TestPlatformHelper.IsMono ?
+            new byte[] { 123, 10, 32, 32, 34, 102, 111, 111, 34, 58, 32, 34, 97, 98, 99, 100, 34, 10, 125 } :
```

Comment:
This isn't mono specific. It is related to Newlines. Use a generic way to handle newlines.